src/plugins/telemetry_management_section/public/components/telemetry_management_section.tsx

```diff
@@ -246,3 +246,7 @@ export class TelemetryManagementSection extends Component<Props, State> {
     });
   };
 }
+
+// required for lazy loading
+// eslint-disable-next-line import/no-default-export
+export { TelemetryManagementSection as default };
```

Comment:
Super NIT: why the `export { X as Y }` instead of the straight `export default TelemetryManagementSection;` format?
```suggestion
export default TelemetryManagementSection;
```